core/src/main/java/org/apache/accumulo/core/conf/AccumuloConfiguration.java

```diff
@@ -209,13 +209,19 @@ static public long getMemoryInBytes(String str) {
         case 'B':
           multiplier = 0;
           break;
+        case '%':
```

Comment:
> If you are worried about certain properties, I could add some text to the description of those properties that fixed memory amounts should be used

That would also assuage my fears. I am not worried about the general misconfiguration -- just users trying to set percentages where it makes no sense.